lib/zendesk_apps_support/manifest.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+module ZendeskAppsSupport
+  class Manifest
+    class << self
+      def private_attr_reader(*names)
+        private
+        attr_reader(*names)
+      end
+    end
+
+    LEGACY_URI_STUB = '_legacy'
+
+    RUBY_TO_JSON = {
```

Comment:
I think this is a reasonably big win, although it will take slightly more work to add properties we can see what we are using at a glance. Also highlights inconsistency like `termsConditionsURL` but `signedUrls`